requirements.txt

```diff
@@ -0,0 +1,12 @@
+asgiref==3.3.1
+Django==3.1.7
+djangorestframework==3.12.2
+pytz==2021.1
+sqlparse==0.4.1
+psycopg2-binary>=2.8.6,<2.9.0
```

Comment:
If not a library, better specify the exact version in order to guaranty a deterministic version in "production"

Reply:
Ah, make sense. I'm so used to "shrink wrap" or lock files from Node, Go, Rust, etc. that I didn't even realize Python doesn't have the equivalent.